presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerPlugin.java

```diff
@@ -18,8 +18,10 @@
 public class SqlServerPlugin
         extends JdbcPlugin
 {
+    public static final String CONNECTOR_NAME = "sqlserver";
```

Comment:
I am looking at `TpchPlugin` or `PostgreSqlPlugin` and it seems we don't have `CONNECTOR_NAME` constant convention. Please inline